scripts/lib.sh

```diff
@@ -106,30 +106,51 @@ install-py-dev() {
     # It places us within top src dir to be at the same path as setup.py
     # This helps when getsentry calls into this script
     cd "${HERE}/.." || exit
+
     echo "--> Installing Sentry (for development)"
     if query-apple-m1; then
         # This installs pyscopg-binary2 since there's no arm64 wheel
         # This saves having to install postgresql on the Developer's machine + using flags
         # https://github.com/psycopg/psycopg2/issues/1286
         pip install https://storage.googleapis.com/python-arm64-wheels/psycopg2_binary-2.8.6-cp38-cp38-macosx_11_0_arm64.whl
```

Comment:
This package was built differently than the other two and we won't be able to work around it. We could get away with it if we installed the postgresql library on the host via brew.